### DISCUSSION TITLE
Proposition for a live zoom

### DISCUSSION THREAD

**Discussion by danymat** [danymat/INSAnonym-utils#53] [Announcements]
Hello everyone!

We are planning to do a live zoom, with the purpose of:

- Showing how to use the insanonym_utils package 
- Explaining how to create your own algorithm with the help of the package
- Doing a Q&A for related INSAnonym competition

We would like to do this web session Wed Jun 09 14:00:00 2021 UTC (16h00 in Paris).
Feel free to comment below if you're interested, and if the hour is suitable for you.

**Comment by mxtla** [comment]
Thanks for the proposition, we are very interested. Could you please send the zoom link?
Thanks

**Reply by beng-git** [reply]
Here is the link. Daniel will be managing the meeting as I have a concurrent meeting.

Topic: DARC live session (Q&A)
Time: Jun 9, 2021 04:00 PM Paris
Join Zoom Meeting
https://zoom.us/j/95642512754?pwd=TWRrL081TGRoU1FSV3Rpa2ZjaDlDQT09
Meeting ID: 956 4251 2754
Passcode: 601417
